tests/pipeline/test_statistical.py

```diff
@@ -400,7 +403,10 @@ def test_correlation_and_regression_with_bad_asset(self):
                 )
 
     def test_require_length_greater_than_one(self):
-        my_asset = Equity(0, exchange="TEST")
+        my_asset = Equity(
+            0,
+            exchange_info=ExchangeInfo('TEST', 'TEST FULL', 'US'),
```

Comment:
Here (and a couple other tests it looks like) the order of `'TEST'` and `'TEST FULL'` look backwards.